Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM fedora
+
+RUN dnf install -y jq skopeo findutils file 'dnf-command(download)'
+
+COPY . /usr/local/bin/
```

Comment:
Can we only add the `BuildSourceImage.sh`?

Reply:
fair. I was thinking there may be detached "plugins", but we can do that later